source/dxgi/dxgi.cpp

```diff
@@ -126,17 +127,23 @@ static void init_reshade_runtime_d3d(T *&swapchain, unsigned int direct3d_versio
 	}
 	else if (direct3d_version == 12)
 	{
-		if (com_ptr<IDXGISwapChain3> swapchain3; SUCCEEDED(swapchain->QueryInterface(&swapchain3)))
+		if (com_ptr<IDXGISwapChain4> swapchain4; SUCCEEDED(swapchain->QueryInterface(&swapchain4)))
```

Comment:
But why? =P Shouldn't SwapChain3 be supported too?

Reply:
Yes, you're right =p reverted back to IDXGISwapChain3 (IDXGISwapChain4 and HDR is not the purpose of this PR)...

Reply:
@crosire do you know why i cannot make the input to work correctly on this UWP title ? I looked at the ICoreInputInterop::SetInputSource method, but i don't know where to link the input to the CoreWindow...